optimizer/src/main/java/com/netease/arctic/optimizer/flink/FlinkReporter.java

```diff
@@ -36,9 +36,18 @@ public class FlinkReporter extends AbstractStreamOperator<Void>
   private static final Logger LOG = LoggerFactory.getLogger(FlinkReporter.class);
   public static final String STATE_JOB_ID = "flink-job-id";
 
+  private static final String JOB_OVERVIEW_REST_API = "/jobs/overview";
+
+  private static final String HIGH_AVAILABILITY_CLUSTER_ID = "high-availability.cluster-id";
+
+  private static final String STATUS_IDENTIFICATION = "status_identification";
+
+  private static final String LAST_MODIFICATION = "last-modification";
+
```

Comment:
`JOB_OVERVIEW_REST_API` `HIGH_AVAILABILITY_CLUSTER_ID` `LAST_MODIFICATION` are useless, we should remove them.